docs/adr/0003-read-data-out-of-account-store-into-app.md

```diff
@@ -0,0 +1,60 @@
+# Reading "Services you have used" Data from the account store into the App
+
+## Summary
+- We will use the AWS SDK to retrieve data directly from the Account User service store
+- We have decided to NOT use AWS API Gateway
+- We have decided to NOT place a lambda between the dynamo and the account management application
+
+## Decision
+
+We will use the AWS SDK to directly query the data store from the express application.
+
+![Architecture diagram showing direct querying of the dynamo](./images/2022-reading-data-out-adopted.png)
+
+We picked this option as it is extremely simple, requiring one dependency added to the app, and no extra AWS components.
+
+As a downside it depends on this application and the dynamo being within the same AWS Account.
```

Comment:
That's not correct - we can set up cross account access to query a DynamoDB table in another account. Take a look at this AWS guide for [cross account access for Dynamo](https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/configure-cross-account-access-to-amazon-dynamodb.html).

Reply:
Which is good! It makes the argument between the options purely a decision between complexity and coupling, and it seems pretty reasonable to choose the low complexity option to speed up implementation at the cost of coupling two services we don't expect to change much

Reply:
thanks for that, didn't know there was cross account dynamo access, very interesting!